src/synology_dsm/api/core/external_usb.py

```diff
@@ -0,0 +1,245 @@
+"""DSM external USB device data."""
+
+from __future__ import annotations
+
+from synology_dsm.api import SynoBaseApi
+from synology_dsm.helpers import SynoFormatHelper
```

Comment:
```suggestion
from synology_dsm.helpers.SynoFormatHelper import bytes_to_readable, megabytes_to_bytes
```

with this we can omit the `SynoFormatHelper.` when using both functions in code

Reply:
I can't get this to work.  When I try it, it get an error: Unable to import 'synology_dsm.helpers.SynoFormatHelper' pylint(import-error)

Reply:
not sure, why this worked in my test (_tbh not sure if it did_ :see_no_evil: ), but you are right, this can not work